### ISSUE TITLE
Add Neural Network data

### ISSUE THREAD

**Issue by einbandi** [jku-vds-lab/projection-space-explorer#13]
Add support the four neural network learning datasets `lr_confmat.csv`, `lr_weights.csv`, `rand_confmat.csv`, and `rand_weights.csv`.

-  `lr_confmat.csv`: contains the confusion matrices and resulting `x` and `y` values for networks trained for 20 epochs with 7 different learning rates. It would be nice to have seven paths with color encoding the learning rate.

-  `lr_weights.csv`: same networks as above, but contains the weight matrices and the resulting projected coordinates `x` and `y`

- `rand_confmat.csv`: contains the confusion matrices for 30 networks trained for 20 epochs, all with the same learning rate. Here categorical coloring does not make sense, as the only thing different between runs was the random initialization.

- `rand_weights.csv`: as above, but weights instead of confusion matrices

For now, we do not necessarily need a detail view, but a confusion matrix view would be nice for all data sets. For the `XX_weights` datasets, you would nee to use the confusion matrices from the associated `XX_confmat` datasets.
